google/api_core/future/polling.py

```diff
@@ -101,13 +103,13 @@ def _blocking_poll(self, timeout=None):
         retry_ = self._retry.with_deadline(timeout)
 
         try:
-            retry_(self._done_or_raise)()
+            retry_(self._done_or_raise)(retry=retry)
```

Comment:
Not sure if `_done_or_raise` is overridden in child classes, but we should also guard this with a `**kwargs` check to be safe.

Reply:
Done